.env.example

```diff
@@ -4,3 +4,4 @@ NEO4J_PASSWORD="password"
 NEO4J_AUTH="${NEO4J_USER}/${NEO4J_PASSWORD}"
 JWT_SECRET="unicourse-jwt-secret"
 CLOUDFLARED_TOKEN=""
+VITE_FIRST_INVITATION_CODE="first_code"
```

Comment:
The prefix is used to make env exposed as `import.meta.env.VITE_SOME_ENV`.

As mentioned in the below comment, this env should not be exposed in client side, and it would be no reason to use `VITE_` anymore. I'll remove the prefix or change it to `UNICOURSE_`.